### PR TITLE
Added Arithmetic Progression in PHP

### LINKED ISSUE #1637: Add PHP and C# code for Arithmetic Progression
I would like to contribute to this issue as a part of GSSoc'20

### PR DESCRIPTION
**Pull request by topcoder69**
Fixes #1637 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Created code of Arithmetic Progression in php
- 

#### Languages Used:
- php
- 

#### Files Added:
- ArithmeticProgression.php
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Arithmetic_Progression/ArithmeticProgression.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
+
```
Comment (jainaman224) [inline Arithmetic_Progression/ArithmeticProgression.php]: Fix file name to ` Arithmetic_Progression.php`

File: Arithmetic_Progression/ArithmeticProgression.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    }
+    /*
+    INPUT : a = 5
+            d = 2
+            n = 7
+    OUTPUT : The term at index 7 is 17
+             Sum is: 77
+    VERIFICATION : The A.P. would be 5,7,9,11,13,15,17...
+    */
+?>
```
Comment (jainaman224) [inline Arithmetic_Progression/ArithmeticProgression.php]: Add a single new line at the end of file.

File: Arithmetic_Progression/ArithmeticProgression.php [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    //input number of terms, i.e, n
+    fscanf(STDIN, "%d\n",$n);
+
+    //progression[] is an array to store AP terms
+    $progression[0] = $a;
+    for ($i = 0; $i < $n; $i++) {
+        $progression[$i + 1] = $progression[$i] + $d;
+    }
+    
+    //output
```
Comment (jainaman224) [inline Arithmetic_Progression/ArithmeticProgression.php]: Add space after `//`.

File: Arithmetic_Progression/ArithmeticProgression.php (hunk 1 above)
Comment (topcoder69) [reply Arithmetic_Progression/ArithmeticProgression.php]: Done!

File: Arithmetic_Progression/ArithmeticProgression.php (hunk 2 above)
Comment (topcoder69) [reply Arithmetic_Progression/ArithmeticProgression.php]: Done

File: Arithmetic_Progression/ArithmeticProgression.php (hunk 3 above)
Comment (topcoder69) [reply Arithmetic_Progression/ArithmeticProgression.php]: Done

**Comment by topcoder69** [comment]
I've updated all the things you mentioned. Please review it! Thanks!

**Comment by raghav-dalmia** [comment]
Looks good to me.

**Comment by raghav-dalmia** [comment]
Squash your commits. Rest looks good to me. I will merge it after that.

**Comment by topcoder69** [comment]
> Squash your commits. Rest looks good to me. I will merge it after that.

Thanks for reviewing. I am not very sure what is that. Should I close this and add a new PR with Updated file?

**Comment by topcoder69** [comment]
Please review it. It has been more than a week.

**Comment by singh-shreya6** [comment]
@topcoder69 There are 2 files in this PR. Please remove the duplicate one. Also please squash your commit post that. You can do it in this PR itself. There is a standard set of git commands for it that you can find on internet easily.

**Comment by singh-shreya6** [comment]
No update from past 27 days on this PR. Closing it. Please reach out if you want to re work on this.

**Pull request closed without merging**